### ISSUE TITLE
Audiocontext was not allowed to start

### ISSUE THREAD

**Issue by shresthapradip** [aws-amplify/amplify-js#6324]
**Motivation**
While trying to use lex, voice conversation doesn't work with warning: audiocontext was not allowed to start.

**To Reproduce**

1. Create new Vue app.
2. Setup amplify and lex correctly. Use amplify-chatbot for chatbot ui.
3. Open app on chrome (safari works as expected)
3. Initiate voice conversation. See it keeps showing listening. 
4. If you go to browser console, you will see the warning Audiocontext was not allowed to start.

**Expected behavior**
No audiocontext error.

**Code Snippet**
````ts
        exports.audioRecorder = function () {
        /**
         * Creates an audio context and calls getUserMedia to request the mic (audio).
         */
        var requestDevice = function requestDevice() {
          if (typeof audio_context === 'undefined') {
            window.AudioContext = window.AudioContext || window.webkitAudioContext;
            audio_context = new AudioContext(); // Error here
          }

          return navigator.mediaDevices.getUserMedia({
            audio: true
          }).then(function (stream) {
            audio_stream = stream;
          });
        };
````

**Additional context**
Only happening on chrome browser.

**Comment by wlee221** [comment]
Hello @shresthapradip, thanks for letting us know! This is currently [being worked on](https://github.com/wlee221/amplify-js/pull/1) along with other bugs we found with the Interactions category. We'll let you know when this makes into the repo. Feel free to reach out if you have any questions :)

**Comment by wlee221** [comment]
Note that you can unblock yourself for now by setting the Chrome flag "Autoplay Policy" to "No user gesture is required", as [explained here](https://developers.google.com/web/updates/2017/09/autoplay-policy-changes). Alternatively you can require a user gesture before loading the chatbot (e.g. adding a button to start the Chatbot instance). We'll keep you updated with the fix 😃

**Comment by wlee221** [comment]
Hi @shresthapradip, this should be fixed on our latest release. Please feel free to open a new issue if problem still persists.
